tests/functional/disaster-recovery/metro-dr-pc/conftest.py

```
@@ -0,0 +1,29 @@
+    if config.MULTICLUSTER.get("multicluster_mode") != constants.MDR_MODE:
+        for item in items.copy():
+            if "disaster-recovery/metro-dr-pc" in str(item.fspath):
+                log.debug(
+                    f"Test {item} is removed from the collected items. Test runs only on MDR clusters"
+                )
+                items.remove(item)
+    else:
+        for item in items.copy():
+            if not config.hci_client_exist():
+                log.debug(
+                    f"Test {item} is removed from the collected items. Test runs only on client MDR clusters"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)